packages/report-server/src/reportBuilder/output/functions/default.ts

```diff
@@ -6,5 +6,5 @@
 import { Row } from '../../types';
 
 export const buildDefault = () => {
-  return (rows: Row[]) => rows;
+  return (rows: Row[]) => ({ data: rows });
```

Comment:
we need to change mapoverlay report server after this one is merged.

Reply:
Hmmm won't this change also affect `PSSS`/`lesmis` as well?